applications/RANSApplication/custom_elements/convection_diffusion_reaction_element_data.h

```diff
@@ -0,0 +1,84 @@
+//    |  /           |
```

Comment:
Note that you are following a different file structure to that one used in the `FluidDynamicsApplication`. We save the data containers in the `custom_utilities` while you are doing so in the `custom_elements` folder. We need to consensus this while merging the apps.

Probably the best solution is to move all the data containers to a `data_containers` folder, which can be within the `custom_elements` folder.

Reply:
I did this because, in FluidApp, custom_utilities folder is kinda exploding :|... I will change it then to `data_containers` under `custom_elements` folder :)

Reply:
This is my point... the `custom_utilities` in there is a mess. The point is that I don't know which is the best way to proceed...
Probably we should do a separated PR in the FluidDynamicsApplication doing the same and you do so in here. Then merging the `custom_elements` folders should be straightforward. Am I wrong?

Reply:
done :)